UI/ImDebugger/ImMemView.cpp

```diff
@@ -319,9 +319,43 @@ void ImMemView::ProcessKeyboardShortcuts(bool focused) {
 		if (ImGui::IsKeyPressed(ImGuiKey_PageUp)) {
 			ScrollWindow(-visibleRows_, GotoModeFromModifiers(false));
 		}
+
+		if(editableMemory_){
```

Comment:
try to follow the style of the surrounding code - for example, we use space after `if` and before `{`, and around operators.